imap_processing/codice/constants.py

```diff
@@ -179,25 +194,6 @@
     9: CoDICECompression.LOSSY_A_LOSSLESS,
 }
 
-# Collapse table ID lookup table for Lo data products
-# The key is the view_id and the value is the ID for the collapse table
-LO_COLLAPSE_TABLE_ID_LOOKUP = {0: 0, 1: 1, 2: 2, 3: 3, 4: 4, 5: 5, 6: 6, 7: 7, 8: 8}
-
-# Collapse table ID lookup table for Hi data products
-# The key is the view_id and the value is the ID for the collapse table
-Hi_COLLAPSE_TABLE_ID_LOOKUP = {
-    0: 8,
-    1: 9,
-    2: 10,
-    3: 0,
-    4: 1,
-    5: 2,
-    6: 4,
-    7: 5,
-    8: 6,
-    9: 7,
-}
```

Comment:
I have found that I can avoid using lookup tables for these because of the way I am defining `DATA_PRODUCT_CONFIGURATIONS`